test/image/mocks/bar_attrs_group_norm.json

```diff
@@ -1,19 +1,24 @@
 {
     "data":[
         {
-            "base":4,
-            "x":[3,2,1,0],
+            "text":0.5,
+            "textposition":"outside",
+            "outsidetextfont": {"size":32},
+            "base":[100,40,25,10],
+            "x":[-50,10,25,40],
             "type":"bar"
         }, {
-            "base":[7,6,5,4],
-            "x":[1,2,3,4],
+            "text":0.5,
```

Comment:
How about we don't rotate the labels at all and let the user decide?

Something like this:
```
    textrotation: {
        valType: 'number',
        dflt: null,
        arrayOk: true,
        role: 'info',
        description: [
            'Rotates the bar text (in clockwise degrees).'
        ].join(' ')
    },
```